server/api/users/users.query.js

```diff
@@ -8,6 +8,12 @@ exports.getAll = function( callback ) {
     });
 }
 
+exports.update = function( data, callback ) {
+    connection.query('UPDATE users SET ? WHERE user_id = ?;', data, function(err, results){
```

Comment:
Maybe security isn't a huge consideration just yet, but are we sanitizing the `data` that's being injected into SQL anywhere?